envs/opence-env.yaml

```diff
@@ -10,7 +10,7 @@ imported_envs:
   - transformers-env.yaml
   - horovod-env.yaml
   - lightgbm-env.yaml
-  - arrow-env.yaml
+  - arrow-env.yaml   #[not s390x or build_type == 'cpu']
```

Comment:
Is this condition required here ? Just for my understanding, Are we trying to run envs/opence-env.yaml on s390x ? Then do we have to mark all other -env.yaml lines in this file with [not s390x] ?

Reply:
We don't usually run opence-env.yaml on s390x. So, you're right! Either we should add [not s390x] for all the envs entries or remove this change although keeping this change won't harm as such.

Reply:
Probably we can avoid having condition here to keep this yaml simple.